templates/web/bexley/waste/services_extra.html

```diff
@@ -0,0 +1,38 @@
+<div class="govuk-grid-row" id="in-cab-logs">
+  [% IF c.user.is_superuser %]
```

Comment:
Who is this block for? Is it for helping debug any issues?

Reply:
Yeah it's for me/us to do debugging. I've added a comment to this template to make it's purpose clear.